.github/workflows/cth-test.yml

```diff
@@ -0,0 +1,88 @@
+name: Conformance test harness
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: false
+        default: latest
+        description: Version of the Conformance Test Harness that has to be tested against
+      ignore_failures:
+        type: boolean
+        default: false
+        description: Return succes even if there are failures?
+        required: false
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        required: false
+        default: latest
+        description: Version of the Conformance Test Harness that has to be tested against.
+      ignore_failures:
+        type: boolean
+        default: false
+        description: Return succes even if there are failures?
+        required: false
+
+jobs:
+  # Run the Conformance Test Harness and upload output as artifacts to GitHub
+  conformance:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Check out the project
+        uses: actions/checkout@v3
+      - name: Install dependencies and run build scripts
+        run: npm ci
+      - name: Start the server in the background
+        run: npm start > server-output.log  &
+      - name: Create the necessary folders
+        run: mkdir -p reports/css
+      - name: Pull the conformance harness docker
+        run: "docker pull solidproject/conformance-test-harness:${{ inputs.version }}"
+      - name: Wait until the server has started
+        run: >
+          curl --output /dev/null --silent --head --fail --retry 30
+          --retry-connrefused --retry-delay 0 -k http://localhost:3000/
+      - name: Create users
+        run: npx ts-node test/deploy/createAccountCredentials.ts http://localhost:3000/ >> test/deploy/conformance.env
+      - name: Set ignore failures option?
+        if: inputs.ignore_failures
+        id: ignore-failures
+        run: echo "::set-output name=flag::--ignore-failures"
```

Comment:
Is there any way for us to know that the CTH failed or do we manually have to check the logs to verify?

Reply:
Yea, check logs or artifacts. I could also just enable continue on error of the github action and not use ignore failures. Then the job will appear failed in the graph but the pipeline will complete.

Another option is to add a comment to the PR?
We can also look into webhooks for Slack/Mattermost

Reply:
Would the job failing prevent the PR from being merged (without having to use admin override)? If not it might be interesting to use that one since the version is currently fixed and we know that we pass. Can always change again in the future if it ends up being an issue.

Comments might be interesting but could get spammy if someone is doing a lot of changes to a PR.

Would not use external methods as those are not useful if someone external is doing a PR.

Reply:
Added the change to fork and started a PR. It reports the failure but I can still merge the PR.
https://github.com/MisterTimn/CommunitySolidServer/pull/83

Also thinking of removing the npm-test dependency... So it fails faster.

Reply:
> Also thinking of removing the npm-test dependency... So it fails faster.

I agree, there is no reason for the CTH to wait.